python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -662,6 +663,63 @@ def testOutputLPEs( self ) :
 				"ieCoreArnold:lpe:testWithAlpha C.*D.*"
 			] ) )
 
+	def testMultipleCameras( self ) :
+		r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"Arnold",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.Batch,
+		)
+
+		for i in range( 3 ):
+			c = r.camera(
+				"testCamera%i"%i,
+				IECoreScene.Camera(
+					parameters = {
+						"projection" : "orthographic"
+					}
+				),
+				r.attributes( IECore.CompoundObject() )
+			)
+			c.transform( imath.M44f().translate( imath.V3f( i, 0, 0 ) ) )
+
+			r.output(
+				"testBeauty%i"%i,
+				IECoreScene.Output(
+					self.temporaryDirectory() + "/beauty%i.exr"%i,
+					"exr",
+					"rgba",
+					{
+						"camera" : "testCamera%i"%i
+					}
+				)
+			)
+
+		r.output(
+			"testDiffuse2",
+			IECoreScene.Output(
+				self.temporaryDirectory() + "/diffuse2.exr",
+				"exr",
+				"diffuse RGBA",
+				{
+					"camera" : "testCamera2"
+				}
+			)
+		)
+
+		r.option( "camera", IECore.StringData( "testCamera2" ) )
+
+		r.render()
+
+		read = GafferImage.ImageReader()
```

Comment:
IECoreArnoldPreview shouldn't have Gaffer dependencies - can we use IECoreImage.ImageReader here instead?